src/mpc-hc/mpcresources/PO/mpc-hc.bg.strings.po

```diff
@@ -2632,6 +2632,11 @@ msgid ""
 "To load subtitles you have to change the video renderer type and reopen the file.\n"
 "- DirectShow: EVR (Custom Presenter), Sync, VMR-9 (renderless), madVR, MPC VR, or Haali VR"
 msgstr ""
+"За да заредите субтитрите трябва да смените видео рендера с някой от посочените и да отворите отново файла.\n"
```

Comment:
This is an old version of the string that was translated. The RM/QT/SW lines should be removed. Just a minor issue. But also, I was expecting more changes to the Bulgarian translation, as the translator said he made several changes?

Reply:
Maybe the merge kept the wrong bit, not sure.  I just forced all transifex translations to get synced, so hopefully all the Bulgarian updates are there, now.